generated/github.ae-anicca-diff-to-api.github.com.deref.json

```diff
@@ -10604,31 +10608,9 @@
       }
     },
     "removed": {
```

Comment:
It appears that a good handful of authorizations / apps endpoints and operations were deprecated.
This looks to be a breaking change.